manticore/platforms/evm.py

```diff
@@ -244,18 +261,38 @@ class EVMException(Exception):
     pass
 
 
-class ConcretizeStack(EVMException):
+class ConcretizeArgument(EVMException):
     '''
-    Raised when a symbolic memory cell needs to be concretized.
+    Raised when a symbolic argument needs to be concretized.
     '''
 
-    def __init__(self, pos, expression=None, policy='MINMAX'):
+    def __init__(self, pos, expression=None, policy='SAMPLED'):
```

Comment:
What are the consequences of changing the policy from MINMAX to SAMPLED?

Reply:
MINMAX selects the maximun and the minimun value.
and SAMPLED takes MIN/MAX _and_ some other values in the middle.